client/nginx.conf

```diff
@@ -15,10 +15,14 @@ server {
 	ssl_certificate /etc/letsencrypt/live/stm.hauke-stieler.de/cert.pem;
 	ssl_certificate_key /etc/letsencrypt/live/stm.hauke-stieler.de/privkey.pem;
 
-	location / {
+	location ~ ^/(en|ja|de)/ {
```

Comment:
This should be `~ ^/en-US/ja/de/`

Reply:
When set here as `~ ^/en-US/` then only English is provided on the site. It depends on production site operation policy.

Reply:
fix uploaded.